webpack/app/scss/_contentchooser.scss

```diff
@@ -72,8 +72,7 @@
   width: 30px;
   height: 30px;
   cursor: pointer;
-  // FIXME - https://github.com/simplesconsultoria/sc.recipe.staticresources/issues/44
-  background-image: url('./img/pb_close.png');
```

Comment:
You must exclude the image itself.

Did you have to copy another resource or was it just this one?